web/src/components/storage/SearchedDeviceMenu.tsx

```diff
@@ -208,6 +209,10 @@ const RemoveEntryOption = ({ device, onClick }: RemoveEntryOptionProps): React.R
     return !onlyToBoot;
   };
 
+  // If the target device cannot be changed, this button will always be disabled and would only
+  // provide redundant information.
+  if (UseOnlyOneOption(device)) return;
```

Comment:
Ah! looks like `UseOnlyOneOption` is not a hook. So, we need better name for it **plus* go for camelCase since it is not a component either. Again, I'm completely aware that this is not part of your work here.